app/src/main/res/values/strings.xml

```diff
@@ -101,6 +101,8 @@
     <string name="about_summary_authors">"© 2016–2021 Tobias Zwick and contributors"</string>
     <string name="about_title_repository">"Source Repository"</string>
     <string name="about_summary_repository">"on GitHub"</string>
+    <string name="about_title_faq">"FAQ"</string>
+    <string name="about_summary_faq">"Please read these first."</string>
```

Comment:
First? Before what?

Reply:
Well it made sense to me (_"do something first"_ - as in _"do this before doing anything else"_. Like how they put it in those "safety warning" pamphlets one gets with user manuals with any gadget / machinery? But it might just be showing my age where [README.1ST](https://en.wikipedia.org/wiki/README.1ST) was the thing.

I will test how to remove the summary completely, then, and get back shortly with update.

Reply:
As suggested, I removed the `about_summary_faq` explanation text, and it seems to work.